src/jquery.data-remote.js

```diff
@@ -177,7 +196,7 @@
         var method = _options.oneAndDone ? 'one' : 'on';
 
         // bind to specific event type
-        $element[method](_options.eventType, function(evt) {
+        $element[method](_options.eventType, debounce(function(evt) {
```

Comment:
How do you wanna approach this organizationally? Do you want two functions, one with debounce and one without, handling different events?